docs/book/src/reference/multi-platform.md

```diff
@@ -0,0 +1,166 @@
+# Platform and Multiple Architecture support
+
+Kubebuilder produces solutions that by default can work on cluster which uses `linux/amd64` platform.
```

Comment:
@aleskandro,

By default we run `make docker-build` :  

```
# docker build --platform linux/arm64 --build-arg=TARGETOS=linux --build-arg=TARGETARCH=arm64 -t ${IMG} .
.PHONY: docker-build
docker-build: test ## Build docker image with the manager.
	docker build -t ${IMG} .
```

The platform is not informed and with the changes of this PR when (the platform is not informed which is the default behaviour) we kept it working as before,see:

```
FROM --platform=${TARGETPLATFORM:-linux/amd64} golang:1.18 as builder
```

When the --flag is not used it will be linux/amd64.

However, please let me know if I missing something?

Reply:
sorted

Reply:
Reopening due to the rollback